SwissTransferCore/Transfer+Extension.swift

```diff
@@ -19,6 +19,8 @@
 import Foundation
 import STCore
 
+extension TransferUi: Identifiable {}
```

Comment:
It looks like by default NSObject have conform to Identifiable but we should be more specific. (We should also check for similar cases in other classes)
```suggestion
extension TransferUi: Identifiable {
    public var id: String {
        uuid
    }
}
```